identity-issuance/src/main/java/com/android/identity/issuance/funke/FunkeProofingState.kt

```diff
@@ -86,18 +108,51 @@ class FunkeProofingState(
                         .replace("\$ISSUER_NAME", issuingAuthorityName)
                         .replace("\$ID_NAME", documentName)
                 }
-                listOf(
-                    EvidenceRequestMessage(
-                        message = message,
-                        assets = emptyMap(),
-                        acceptButtonText = "Continue",
-                        rejectButtonText = "Cancel"
-                    )
-                )
-            } else if (!notificationPermissonRequested) {
-                listOf(
-                    EvidenceRequestNotificationPermission(
-                    permissionNotGrantedMessage = """
+                listOf(EvidenceRequestMessage(
+                    message = message,
+                    assets = emptyMap(),
+                    acceptButtonText = "Continue",
+                    rejectButtonText = "Cancel"
+                ))
+            } else if (txCode == null &&
+                credentialOffer is Openid4VciCredentialOfferPreauthorizedCode &&
+                credentialOffer.txCode != null) {
+                val message = "<p>" +
+                        credentialOffer.txCode!!.message.replace("<", "&lt;") +
```

Comment:
I see we have a htmlEscape function in AdminServlet.kt. Rather than having different hacks in different areas of the code, could we move that function to a common library/utility and reuse it here? (maybe something in identity/src/commonMain/kotlin/com/android/identity/util ?)

Reply:
Done